### PR TITLE
Fixed an issue where the shopping cart remained empty due to a header…

### PR DESCRIPTION
**Pull request by younes-dro**
… modification error.

Add file : languages\ets_q_n_a.pot  pulled with 'SVN UP' command

### CODE REVIEW

File: includes/ets_user_qa_function.php [hunk 1]
```diff
@@ -1,193 +1,194 @@
+<?php
```
Comment (geek92sunny) [inline includes/ets_user_qa_function.php:1]: The blank space on the start of the file was the only change we did here?

File: languages/ets_q_n_a.pot [hunk 2]
```diff
@@ -0,0 +1,163 @@
+# Copyright (C) 2023 ExpressTech Software Solutions Pvt. Ltd.
```
Comment (geek92sunny) [inline languages/ets_q_n_a.pot]: Discard this POT, this file should have same name as the text domain.

File: includes/ets_user_qa_function.php [hunk 3]
```diff
@@ -196,380 +197,411 @@ public function ets_ask_qustion_tab() {
...
 
-					<?php 
-					if(!empty($value['answer'])){?>
+					<?php
+					if ( ! empty( $value['answer'] ) ) {
+						?>
 						<tr>
-							<td class="ets-question-title"><p><?php echo __('Answer','ets_q_n_a'); ?>:</p></td>
-							<td colspan="2"><p> <?php echo $value['answer'];?></p></td> 
+							<td class="ets-question-title"><p><?php echo __( 'Answer', 'ets_q_n_a' ); ?>:</p></td>
```
Comment (geek92sunny) [inline includes/ets_user_qa_function.php]: Please update text domain in translation func here.

File: includes/ets_user_qa_function.php [hunk 4]
```diff
@@ -196,380 +197,411 @@ public function ets_ask_qustion_tab() {
...
 						</tr> 
-						<?php 
-					} else { ?>
+						<?php
+					} else {
+						?>
 						<tr>
-							<td class="ets-question-title"><p><?php echo __('Answer','ets_q_n_a'); ?>:</p></td>
-							<td colspan="2" class="ets-no-answer"><h6><p><i><?php echo __("Answer awaiting",'ets_q_n_a');?>...</i></p></h6></td>	
+							<td class="ets-question-title"><p><?php echo __( 'Answer', 'ets_q_n_a' ); ?>:</p></td>
```
Comment (geek92sunny) [inline includes/ets_user_qa_function.php]: Please update text domain name here too.

File: readme.txt [hunk 5]
```diff
@@ -85,4 +85,7 @@ No configuration needed! just install and it will start working from there, you
 - Update: Added support for WordPress 6.5 compatibility. Tested successfully.
 
 = 1.2.6 =
-- Dev: Added the shortcode [display_qa_list]
\ No newline at end of file
+- Dev: Added the shortcode [display_qa_list]
+
+= 1.2.7 =
```
Comment (geek92sunny) [inline readme.txt:90]: Rephrase changelog  "Fixed the blockage in the AJAX requests."

**Review by geek92sunny — CHANGES_REQUESTED** [review]
A question and some change requests, version these as 1.2.8.

File: includes/ets_user_qa_function.php (hunk 1 above)
Comment (younes-dro) [reply includes/ets_user_qa_function.php:1]: Yes

**Review by geek92sunny — CHANGES_REQUESTED** [review]
We already have a `product-questions-answers-for-woocommerce.pot` file under language directory, please update that and discard the new POT file you created.

File: readme.txt (hunk 5 above)
Comment (geek92sunny) [reply readme.txt:90]: Please rephrase the 1.2.7 as mentioned above.

**Comment by geek92sunny** [comment]
Please do the pending changes and update the version to 1.2.8 @younes-dro

File: includes/ets_user_qa_function.php (hunk 3 above)
Comment (younes-dro) [reply includes/ets_user_qa_function.php]: > Please update text domain in translation func here.

Already updated.
dont find any text domain : 'ets_q_n_a' in the plugin files

**Comment by geek92sunny** [comment]
Since we also changed the text domain in some places, can you please change the version to 1.2.8 ?

The changelog will be 
= 1.2.6 =
\- Dev: Replaced old text domain to some places

**Comment by younes-dro** [comment]
> Since we also changed the text domain in some places, can you please change the version to 1.2.8 ?
> 
> The changelog will be = 1.2.6 = - Dev: Replaced old text domain to some places

@geek92sunny you mean 1.2.8 NOT 1.2.6 ?

**Review by geek92sunny — APPROVED** [review]
All set!

**Pull request merged**